### ISSUE TITLE
kata release 2.4.3 creat pod has been in the ContainerCreating state with sriov ib infiniband.

### ISSUE THREAD

**Issue by fu7100** [kata-containers/kata-containers#4954]
I am using kata runtime on openshift 4.9 and using sriov's ib infiniband, I want to pass vf through to kata pod, but when i create a pod, it keeps in containercreating state, finally describe pod log shows as below

![图片](https://user-images.githubusercontent.com/59348973/185299598-30aed8e2-ff57-41e8-8be7-77b91b3be1f9.png)

The ib card is installed on the ocp worker0 host and sriov virtualization is configured through ocp `sriov operator`, vf number is 10
![图片](https://user-images.githubusercontent.com/59348973/185299861-210cb924-9131-42eb-8e58-52d5db9d7c72.png)

**Comment by fidencio** [comment]
@fu7100, if you're using kata-deploy on OpenShift, I think you may end up hitting some issues related to SELinux.
Would you mind to double check that's not the case?

Ideally, with OpenShift, you should be able to take advantage of the OpenShift sandboxed containers, which is the Red Hat solution which uses Kata Containers.

**Comment by fu7100** [comment]
@fidencio  Yes, I did use the redhat solution and I turned off selinux, but still the same.
Do you need me to provide some log data?, I can collect.

**Comment by fidencio** [comment]
Let me loop Red Hat folks in here:
@gkurz @bpradipt @snir911 @ariel-adam
